Makefile

```diff
@@ -23,6 +25,15 @@ LOCAL_ARTIFACT_MIRROR_IMAGE ?= proxy.replicated.com/anonymous/replicated/embedde
 METADATA_K0S_BINARY_URL_OVERRIDE =
 METADATA_KOTS_BINARY_URL_OVERRIDE =
 METADATA_OPERATOR_BINARY_URL_OVERRIDE =
+
+ifeq ($(ARCH),amd64)
+ifeq ($(K0S_VERSION),v1.29.9+k0s.0)
+K0S_BINARY_SOURCE_OVERRIDE = https://embedded-cluster-assets.s3.amazonaws.com/k0s-v1.29.9%2Bk0s.0-amd64
```

Comment:
contains this patch

https://github.com/k0sproject/k0s/compare/main...emosbaugh:k0s:fix-forceUpgrade-nil-default?expand=1

Reply:
do we need this if my PR gets merged?

Reply:
What would change this about your pr?

Reply:
Well, with the new upgrade path (job in cluster, I mean), we don't need this patch
however, that won't fix the issue when upgrading from pre-1.8